### DISCUSSION TITLE
询问最底部这个标识如何生成？

### DISCUSSION THREAD

**Discussion by Asakushen** [anzhiyu-c/hexo-theme-anzhiyu#169] [Q&A] [answered]
![image](https://github.com/anzhiyu-c/hexo-theme-anzhiyu/assets/74193315/54103f08-10d7-4359-9f32-04f57c90295b)
如图感觉很酷，是通过shields.io生成的吗？如何插入图标呢？求大佬解答！

**Comment by sun0225SUN** [comment, marked as answer]
https://akilar.top/posts/e87ad7f8/

可以参考此教程，若图标库没有自己想要的图标，可以将图片转成 base64 格式（教程最下面）

**Reply by Asakushen** [reply]
哇大佬，谢谢解答！晚上才看完你的"第二大脑“博客内容，没想到居然在这里遇见了！

**Reply by Asakushen** [reply]
自己摸索半天 主要卡在再来必须是svg转的base64才行 我一直再用png转，而且分辨率不能太大，五分钟前我刚刚解决这个问题，但还是非常感谢解答！
